openreview/conference/builder.py

```diff
@@ -2805,8 +2812,13 @@ def set_registration_stage(self, committee_id, name = 'Registration', start_date
         reviewer_instructions = instructions if instructions else default_instructions
         self.registration_stages.append(RegistrationStage(committee_id, name, start_date, due_date, additional_fields, reviewer_instructions))
 
-    def set_bid_stage(self, committee_id, start_date = None, due_date = None, request_count = 50, score_ids = [], instructions = False):
-        self.bid_stages.append(BidStage(committee_id, start_date, due_date, request_count, score_ids, instructions))
+    @deprecated(version='1.6.0')
```

Comment:
I added this back because tests are using it, but I can also change the tests to use set_bid_stages() instead, what do you think @melisabok?

Reply:
ok, let's keep it for now